pkg/apis/v1alpha1/serviceexport.go

```diff
@@ -88,3 +92,134 @@ type ServiceExportList struct {
+	// * "Failed"
+	//
+	// Possible reasons for this condition to be Unknown are:
+	//
+	// * "Pending"
+	//
+	// Controllers may raise this condition with other reasons,
+	// but should prefer to use the reasons listed above to improve
+	// interoperability.
+	ServiceExportConditionExported = "Exported"
+
+	// ServiceExportReasonExported is used with the "Exported" condition
+	// when the condition is True.
+	ServiceExportReasonExported = "Exported"
+
+	// ServiceExportReasonPending is used with the "Exported" condition
+	// when the service is going to be exported.
+	ServiceExportReasonPending = "Pending"
+
+	// ServiceExportReasonFailed is used with the "Exported" condition
+	// when the service failed to be exported.
+	ServiceExportReasonFailed = "Failed"
+)
+
+const (
+	// ServiceExportConditionConflicted indicates that the controller was unable
+	// to resolve conflict for a ServiceExport. This condition must be at
+	// least raised on the conflicting ServiceExport and is recommended to
+	// be raised on all on all the constituent `ServiceExport`s if feasible.
+	//
+	//
+	// Possible reasons for this condition to be true are:
+	//
+	// * "PortConflict"
+	// * "TypeConflict"
+	// * "SessionAffinityConflict"
+	// * "SessionAffinityConfigConflict"
+	// * "AnnotationsConflict"
+	// * "LabelsConflict"
+	//
+	// When multiple conflicts occurs the above reasons may be combined
+	// using commas.
+	//
+	// Possible reasons for this condition to be False are:
+	//
+	// * "NoConflicts"
+	//
+	// Controllers may raise this condition with other reasons,
+	// but should prefer to use the reasons listed above to improve
+	// interoperability.
+	ServiceExportConditionConflicted = "Conflicted"
+
+	// ServiceExportReasonPortConflict is used with the "Conflicted" condition
+	// when the exported service have a conflict related to port configuration.
+	// This includes when ports on resulting imported services would have
+	// duplicated names (including unnamed/empty name) or duplicated
+	// port/protocol pairs.
+	ServiceExportReasonPortConflict = "PortConflict"
+
+	// ServiceExportReasonTypeConflict is used with the "Conflicted" condition
+	// when the exported service have a conflict related to type.
+	ServiceExportReasonTypeConflict = "TypeConflict"
+
+	// ServiceExportReasonSessionAffinityConflict is used with the "Conflicted"
+	// condition when the exported service have a conflict related to session affinity.
+	ServiceExportReasonSessionAffinityConflict = "SessionAffinityConflict"
+
+	// ServiceExportReasonSessionAffinityConfigConflict is used with the
+	// "Conflicted" condition when the exported service have a conflict related
+	// to session affinity config.
+	ServiceExportReasonSessionAffinityConfigConflict = "SessionAffinityConfigConflict"
+
+	// ServiceExportReasonLabelsConflict is used with the "Conflicted"
+	// condition when the exported service have a conflict related to labels.
+	ServiceExportReasonLabelsConflict = "LabelsConflict"
```

Comment:
I think this should be treated similar as `PortConflict`.

```suggestion
	ServiceExportReasonLabelConflict = "LabelConflict"
```